wordpress-mobile/WordPress-iOS
# Update site header UI

After updating the topic page header in #14525 , we are now giving the site header a fresh new look as well.

To test:
1. Go to reader 
2. Tap on a site name in a post card 
3. See the new site header

Before             |  After
:-------------------------:|:-------------------------:
![](https://user-images.githubusercontent.com/1335657/88749923-35751d80-d109-11ea-9ab0-c46b4662d708.jpeg)  |  ![](https://user-images.githubusercontent.com/1335657/88749972-550c4600-d109-11ea-808c-5a5bbed12b87.png)

### With a longer site description 
![Screen Shot 2020-07-28 at 6 36 31 PM](https://user-images.githubusercontent.com/1335657/88750179-cc41da00-d109-11ea-939b-a645ffa8f682.png)


PR submission checklist:
